app/views/includes/paye/PayeMainView.scala.html

```diff
@@ -28,7 +28,9 @@
 
 @(title: String,
 userLoggedIn: Boolean = true,
+mainViewParameters: MainViewParameters,
```

Comment:
The next step here is, you should be able to create a default version of this: 
`mainViewParameters: MainViewParameters = MainViewParameters(true, true)` 
The above line might change depending on how you code the default behaviour in MainViewParameters.scala